From: Andrej Shadura <andrew.shadura@collabora.co.uk>
Date: Mon, 6 Nov 2017 10:22:03 +0100
Subject: build-recipe-dsc: don't rebuild the source package

If we use build suffixes, rebuilding the source package will
result in the build suffix being included into the source package,
which we'd rather avoid.

Signed-off-by: Andrej Shadura <andrew.shadura@collabora.co.uk>
Forwarded: https://github.com/openSUSE/obs-build/pull/459
Last-Update: 2018-08-30
---
 build-recipe-dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build-recipe-dsc b/build-recipe-dsc
index d748c4e..9243776 100644
--- a/build-recipe-dsc
+++ b/build-recipe-dsc
@@ -95,7 +95,7 @@ dsc_build() {
     # this allows the build environment to be manipulated
     # and alternate build commands can be used
     DSC_BUILD_CMD="$(queryconfig --dist "$BUILD_DIST" --archpath "$BUILD_ARCH" --configdir "$CONFIG_DIR" substitute dsc:build_cmd)"
-    test -z "$DSC_BUILD_CMD" && DSC_BUILD_CMD="dpkg-buildpackage -us -uc"
+    test -z "$DSC_BUILD_CMD" && DSC_BUILD_CMD="dpkg-buildpackage -us -uc -b"
     if test -e $buildroot/$TOPDIR/SOURCES/build.script ; then
 	echo "Sourcing build.script to build - it should normally run 'dpkg-buildpackage -us -uc'"
 	DSC_BUILD_CMD="source $TOPDIR/SOURCES/build.script"
